### DISCUSSION TITLE
Guitar Rig 5

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#29] [General]
# Guitar Rig 5

Check if Guitar Rig 5 is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/3bc9bd8d-fcc2-4f04-9a63-e8f1f9a73009

**Comment by OpenSource03** [comment]
Hello.

Would you be able to test this for us? The app seems to be behind paywall, so we can not easily tell if it's native or not.

Most likely this will be one of those x64 emulated apps with poor performance, but I am unable to verify at this moment.
